### PR TITLE
use minitest for the storage samples

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**
partially addresses https://github.com/GoogleCloudPlatform/ruby-docs-samples/issues/479

### CODE REVIEW

**Comment by TheRoyalTnetennba** [comment]
kokoro failures are unrelated to this PR

**Comment by TheRoyalTnetennba** [comment]
> I'm pretty sure I've brought this up before: I think it would be good to add `Gemfile.lock` to `.gitignore`.

🤷‍♂ it was brought up in https://github.com/GoogleCloudPlatform/ruby-docs-samples/pull/408. I'd prefer to leave it as is for now, revisiting the topic if/when we move samples to the library repos.

**Comment by TheRoyalTnetennba** [comment]
The only way I could repro the issue is by installing google-cloud-storage from the current version on master rather than the released version. Could you see if reinstalling google-cloud-storage works?

**Comment by TheRoyalTnetennba** [comment]
@quartzmo could take another look at this?

File: storage/acceptance/acls_test.rb [hunk 1]
```diff
@@ -0,0 +1,151 @@
+require_relative "helper"
```
Comment (dazuma) [inline storage/acceptance/acls_test.rb]: Please add the copyright header to all the test files. `Copyright 2020 Google LLC` etc.

**Comment by TheRoyalTnetennba** [comment]
CI is failing due to https://github.com/GoogleCloudPlatform/ruby-docs-samples/issues/496 and looks like the vision tests need to be updated

**Pull request merged**